package.json

```diff
@@ -8,8 +8,20 @@
     "postinstall": "lerna bootstrap --ci --force-local",
     "postpublish": "lerna exec -- npm install --package-lock-only && git commit -a -m 'Bump package-lock' && git push"
   },
-  "devDependencies": {},
   "dependencies": {
     "lerna": "^3.22.1"
+  },
+  "devDependencies": {
```

Comment:
is this stuff needed?

Reply:
For my editor, VSCode, all integrated linting fails if the packages are not installed at the top level. I'd leave this as uncommitted, but then `npm ci` removes